hudi-spark/src/main/scala/org/apache/hudi/DataSourceOptions.scala

```diff
@@ -108,6 +108,15 @@ object DataSourceReadOptions {
     */
   val END_INSTANTTIME_OPT_KEY = "hoodie.datasource.read.end.instanttime"
 
+  /**
+    * If use the end instant schema when incrementally fetched data to.
+    *
+    * Default: false (use latest instant schema)
+    *
+    */
+  val INCREMENTAL_READ_SCHEMA_USE_ENDINSTANT_OPT_KEY = "hoodie.datasource.read.schema.use_endInstant"
```

Comment:
Changing to `INCREMENTAL_READ_SCHEMA_USE_END_INSTANTTIME_OPT_KEY = "hoodie.datasource.read.schema.use.end.instanttime"`?

Reply:
make sense. have resolved